css/style.css

```diff
@@ -0,0 +1,1248 @@
+  box-sizing: inherit;
+}
+
+body,
+button,
+input,
+textarea {
+  font-family: "helvetica neue", arial, sans-serif;
+  font-size: 16px;
+}
+
+body {
+  color: #2e2e2e;
+  background-color: #fff;
+  margin: 0 auto;
+  text-align: center;
+}
+
+p,
+button,
+textarea,
+input {
+  color: #fff;
+}
+
+h1 {
+  font-size: 2em;
+  font-weight: lighter;
+  line-height: 1.3;
+}
+
+h2 {
+  font-size: 2.369em;
+  font-weight: bold;
+}
+
+h3 {
+  font-size: 1.777em;
+  font-weight: bold;
+}
+
+h4 {
+  font-size: 0.9em;
+}
+
+button {
+  background-color: #fd6632;
+  border: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+button:active, a:active {
+  opacity: 0.8;
+}
+
+button:hover {
+  cursor: pointer;
+}
+
+img {
+  display: block;
+  max-width: 100%;
+  height: auto;
+  opacity: 0.7;
+}
+
+input,
+textarea {
+  display: block;
+  font-weight: bold;
+  background-color: #be4c25;
+  border: none;
+}
+
+li {
+  list-style-type: none;
+}
+
+progress[value] {
+  -webkit-appearance: none;
+  appearance: none;
+  height: 1.2em;
+  width: 100%;
+}
+
+/*
+MODIFIERS
+*/
+
+.white-color {
+  color: #fff;
+}
+
+.circle {
+  border-radius: 100%;
+}
+
+/*
+ICON FONTS
+*/
+
+@font-face {
+  font-family: 'ico-fonts';
+  src:url('/css/fonts/ico-fonts.eot?ghgy45');
+  src:url('/css/fonts/ico-fonts.eot?#iefixghgy45') format('embedded-opentype'),
+    url('/css/fonts/ico-fonts.ttf?ghgy45') format('truetype'),
+    url('/css/fonts/ico-fonts.woff?ghgy45') format('woff'),
+    url('/css/fonts/ico-fonts.svg?ghgy45#ico-fonts') format('svg');
+  font-weight: normal;
+  font-style: normal;
+}
+
+[class^="icon-"], [class*=" icon-"] {
+  font-family: 'ico-fonts';
+  speak: none;
+  font-style: normal;
+  font-weight: normal;
+  font-variant: normal;
+  text-transform: none;
+  line-height: 1;
+
+
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+[class^="icon-"]:link, [class*=" icon-"]:link,
+[class^="icon-"]:visited, [class*=" icon-"]:visited {
+  color: #fff;
+}
+
+.icon-gear:before {
+  content: "\e601";
+}
+.icon-envelope:before {
+  content: "\e602";
+}
+.icon-lab:before {
+  content: "\e600";
+}
+.icon-arrow-down:before {
+  content: "\e610";
+}
+.icon-google-plus:before {
+  content: "\e603";
+}
+.icon-facebook:before {
+  content: "\e604";
+}
+.icon-twitter:before {
+  content: "\e605";
+}
+.icon-dribbble:before {
+  content: "\e606";
+}
+.icon-linkedin2:before {
+  content: "\e607";
+}
+.icon-menu:before {
+  content: "\e60f";
+}
+.icon-drop:before {
+  content: "\e60e";
+}
+.icon-plus:before {
+  content: "\e608";
+}
+.icon-chevron-left:before {
+  content: "\e609";
+}
+.icon-chevron-right:before {
+  content: "\e60a";
+}
+.icon-brush:before {
+  content: "\e60b";
+}
+.icon-phone:before {
+  content: "\e60c";
+}
+.icon-pin:before {
+  content: "\e60d";
+}
+
+
+/*
+SECTION HEADERS
+*/
+
+.team-section-header,
+.client-section-header,
+.section-header,
+.sub-section-header,
+.portfolio-section-header,
+.contact-section-header {
```

Comment:
With a selector string of this length, I'd probably consider making a re-usable class that describes these sorts of sections and then apply it to the headings. This isn't a problem yet, but is the start of some code smell.